mne/io/cnt/cnt.py

```diff
@@ -491,16 +497,6 @@ class RawCNT(BaseRaw):
         if either contain a bad channel make channel bad.
         Defaults to ``'auto'``.
     %(preload)s
-    stim_channel : bool | None
-        Add a stim channel from the events. Defaults to None to trigger a
-        future warning.
-
-        .. warning:: This defaults to True in 0.18 but will change to False in
-                     0.19 (when no stim channel synthesis will be allowed)
-                     and be removed in 0.20; migrate code to use
-                     :func:`mne.events_from_annotations` instead.
-
-        .. versionadded:: 0.18
```

Comment:
I'm surprised this did not get picked-up by `numpydoc`. The argument is missing from the class signature!

Reply:
We only have numpydoc check things that are in the public API, and `RawCNT` is not